src/MenuItem/MenuItem.js

```diff
@@ -297,6 +297,7 @@ class MenuItem extends Component {
         ref="listItem"
         rightIcon={rightIconElement}
         style={mergedRootStyles}
+        hoverColor={this.context.muiTheme.menuItem.hoverColor}
```

Comment:
The other properties are alphabetically sorted. Might be a coincidence for all I know.